app/controllers/stats/data/base_controller.rb

```diff
@@ -24,6 +24,8 @@ class Stats::Data::BaseController < ApplicationController
   ##
   ##~ @parameter_service_id = {:name => "service_id", :description => "ID of the service.", :dataType => "int", :required => true, :paramType => "path", :threescale_name => "service_ids"}
   ##
+  ##~ @parameter_backend_api_id_by_id_name = { :name => "backend_api_id", :description => "ID of the Backend.", :dataType => "int", :required => true, :paramType => "path" }
```

Comment:
wow, I see that there is some repetition between `Stats::Data::BaseController` (this class) and `Admin::Api::BaseController`:
https://github.com/3scale/porta/blob/c67d5d8227a1f64c0ff2c0d91f5ea39a00cb9fb9/app/controllers/admin/api/base_controller.rb#L14-L15
https://github.com/3scale/porta/blob/c67d5d8227a1f64c0ff2c0d91f5ea39a00cb9fb9/app/controllers/admin/api/base_controller.rb#L132

but definitely not your fault 👍